.github/workflows/deploy-to-test-and-live.yml

```diff
@@ -0,0 +1,78 @@
+# Github action to deploy to Pantheon
+name: Deploy to Pantheon Test and Live
+
+on:
+  release:
+    types: [published]  # Trigger when a release is published
+
+jobs:
+  deploy-to-test:
+    environment: Test
+    permissions:
+      contents: read
+      deployments: write
+      pull-requests: read
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.release.tag_name }}  # Checkout the release tag (e.g., v1.0.0)
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: "7.4"
+
+      - name: Install Terminus
+        uses: pantheon-systems/terminus-github-actions@v1
+        with:
+          pantheon-machine-token: ${{ secrets.PANTHEON_MACHINE_TOKEN }}
+          disable-cache: true
+
+      - name: Prepare deployment message
+        run: |
+          RELEASE_BODY=$(echo "${{ github.event.release.body }}")
+          echo "DEPLOY_TEST_MESSAGE<<EOF" >> $GITHUB_ENV
+          echo "$RELEASE_BODY" >> $GITHUB_ENV
+          echo "EOF" >> $GITHUB_ENV
+
+      - name: Trigger Test Deploy
+        run: |          
+          terminus env:deploy ${{ secrets.PANTHEON_SITE_NAME }}.test --note "${{ env.DEPLOY_TEST_MESSAGE }}"
+
+  deploy-to-live:
+    environment: Live # This environment requries manual approval
```

Comment:
Is there something in this YML config that makes the deploy to live require manual approval, or is that configured elsewhere?

Reply:
Configured as [reviewers the environment Live](https://github.com/eGovPDX/portlandor/settings/environments/6367566890/edit). This is the only part that's not tested because I was using public-alerts as the test bed but that repo is not public. Only public repo's can have reviewers for our GitHub plan.